lib/community/widgets/post_card_metadata.dart

```diff
@@ -30,87 +30,91 @@ class PostCardMetaData extends StatelessWidget {
+                            size: 20.0,
+                            color: voteType == VoteType.up
+                                ? upVoteColor
+                                : voteType == VoteType.down
+                                    ? downVoteColor
+                                    : readColor),
+                        padding: 2.0,
+                      ),
+                      const SizedBox(width: 10.0),
+                      IconText(
+                        textScaleFactor: state.contentFontSizeScale.textScaleFactor,
+                        icon: Icon(
+                          /*unreadComments != 0 && unreadComments != comments ? Icons.mark_unread_chat_alt_rounded  :*/ Icons.chat,
+                          size: 15.0,
+                          color: /*unreadComments != 0 && unreadComments != comments ? theme.primaryColor :*/
+                              readColor,
+                        ),
+                        text: /*unreadComments != 0 && unreadComments != comments ? '+${formatNumberToK(unreadComments)}' :*/
+                            formatNumberToK(comments),
+                        textColor: /*unreadComments != 0 && unreadComments != comments ? theme.primaryColor :*/
+                            readColor,
+                        padding: 5.0,
+                      ),
+                      const SizedBox(width: 10.0),
+                      IconText(
+                        textScaleFactor: state.contentFontSizeScale.textScaleFactor,
+                        icon: Icon(
+                          hasBeenEdited ? Icons.refresh_rounded : Icons.history_rounded,
+                          size: 15.0,
+                          color: readColor,
+                        ),
+                        text: formatTimeToString(dateTime: published.toIso8601String()),
+                        textColor: readColor,
+                      ),
+                      const SizedBox(width: 8.0),
+                    ],
                   ),
-              ],
-            ),
-            if (useCompactView)
-              Icon(
-                saved ? Icons.star_rounded : null,
-                color: saved ? savedColor : null,
-                size: 22.0,
-                semanticLabel: saved ? 'Saved' : '',
+                  if (hostURL != null)
+                    Padding(
+                      padding: const EdgeInsets.only(left: 2.0),
+                      child: IconText(
+                        textScaleFactor: state.contentFontSizeScale.textScaleFactor,
+                        icon: Icon(
+                          Icons.public,
+                          size: 15.0,
+                          color: readColor,
+                        ),
+                        text: Uri.parse(hostURL!).host.replaceAll('www.', ''),
```

Comment:
In the crazy unlikely chance that `www` is contained elsewhere in the URL, do you want to use `replaceFirst` here?